### ISSUE TITLE
Proxy Config for idstools

### ISSUE THREAD

**Issue by TOoSmOotH** [Security-Onion-Solutions/securityonion#2587]
I don't have any file called: /opt/so/rules/nids/all.rules
I only have: /opt/so/rules/nids/local.rules

Thusly, Suricata is not generating any alerts, since it has no rules. 

Any advice on getting those in place? I selected ETOPEN during the manager node setup, but it doesn't seem to have pulled any rules. 

I think it might be related to me using a proxy. Using so-rule-update seems to fail, timing out on fetching rules. In the old version of SO, you had to add a line in the SO config to add the -W option to pulled pork, in order for it to work via proxy. Is there something like that going on here?

_Originally posted by @w4rc0n in https://github.com/Security-Onion-Solutions/securityonion/discussions/2583_

**Comment by edwardrixon** [comment]
I have been looking into this and have attempted the following so far without success.

I have added busy box compatible proxy settings into the docker with the following commands but they do not persist on stop start of the container.

- docker -it -u 0 exec so-idstools /bin/bash
- vi /etc/profile
- I have then added the following without success so far;
- export http_proxy=<proxy address>
- export HTTP_PROXY=<proxy address>
- http_proxy=<proxy address>
- HTTP_PROXY=<proxy address>

Hope this helps with finding a solution to this as I am in the same situation. I have managed to do all my soup updates by configuring my git, docker, and env's for centos7 but I cannot get this to function.
